File: .sdk/tm/rb/utility/make_context.rb
# PassantenfrequenzenZuerich SDK utility: make_context
require_relative '../core/context'
module PassantenfrequenzenZuerichUtilities
  MakeContext = ->(ctxmap, basectx) {
    PassantenfrequenzenZuerichContext.new(ctxmap, basectx)
  }
end

File: .sdk/tm/py/pkg/utility/make_context.py
# PassantenfrequenzenZuerich SDK utility: make_context

from projectname_sdk.core.context import PassantenfrequenzenZuerichContext


def make_context_util(ctxmap, basectx):
    return PassantenfrequenzenZuerichContext(ctxmap, basectx)
